platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r812/ProblemThresholdCrossVersionTest.groovy

```diff
@@ -133,12 +133,41 @@ class ProblemThresholdCrossVersionTest extends ToolingApiSpecification {
         then:
         def problems = listener.problems
         problems.size() == THRESHOLD_DEFAULT_VALUE + differentProblemCount
-        validateFirstNProblems(THRESHOLD_DEFAULT_VALUE, problems)
+        validateNToMProblems(0, THRESHOLD_DEFAULT_VALUE, problems)
         def problemSummariesEvent = listener.summariesEvent as ProblemSummariesEvent
         def summaries = problemSummariesEvent.problemSummaries
         summaries.size() == 1
     }
 
+    def "Two problem ids exceed threshold"() {
```

Comment:
💅 No action needed, but just a meta comment.  I'd prefer to see test methods communicate the expectation or stimulus/response that they are verifying.  This doesn't make it clear what this test is verifying - if it failed, I'd have no idea what was broken, much less how I might have affected it.